lib/core/src/lib/pipes/moment-datetime-utc-to-local.pipe.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Pipe, PipeTransform } from '@angular/core';
+import moment, { Moment } from 'moment';
+
+@Pipe({ name: 'adfMomentDateTimeUtcToLocal' })
```

Comment:
we are moving away from the moment, maybe you could use the https://date-fns.org/ instead? it is already used by the ADF components